app/assets/stylesheets/govuk_publishing_components/components/_layout-super-navigation-header.scss

```diff
@@ -472,17 +514,40 @@ $after-button-padding-left: govuk-spacing(4);
     .gem-c-layout-super-navigation-header__navigation-top-toggle-button-inner {
       border-color: $button-pipe-colour;
 
+      &.gem-c-layout-super-navigation-header__navigation-top-toggle-button-inner--blue-background {
+        border-color: $pale-blue-colour;
+      }
+
       @include govuk-media-query($from: 360px) {
         &::before {
           @include chevron(govuk-colour("white"), true);
         }
       }
     }
+
     // stylelint-enable max-nesting-depth
   }
+}
+
+.gem-c-layout-super-navigation-header__navigation-top-toggle-button--large-navbar {
+  height: $large-navbar-height;
+}
 
+.gem-c-layout-super-navigation-header__navigation-top-toggle-button.gem-c-layout-super-navigation-header__navigation-top-toggle-button--blue-background,
+.gem-c-layout-super-navigation-header__navigation-top-toggle-button {
   // Open button modifier
   &.gem-c-layout-super-navigation-header__open-button {
+    &::before {
```

Comment:
There's a white block caused by this pseudo element when the button is focused and the menu is open. Not sure if that's part of the design?  
<img width="325" alt="Screenshot 2023-10-03 at 14 10 51" src="https://github.com/alphagov/govuk_publishing_components/assets/5007934/95416b1c-f3f7-4071-a96a-ed71ecea10fe">

Reply:
Have removed this